### ISSUE TITLE
ModuleNotFoundError: No module named 'triegex'

### ISSUE THREAD

**Issue by 5j9** [ZhukovAlexander/triegex#2]
I installed the package from pypi using pip:

```
$ pip install triegex
Collecting triegex
  Downloading triegex-0.0.2.tar.gz
Building wheels for collected packages: triegex
  Running setup.py bdist_wheel for triegex ... done
  Stored in directory: C:\Users\a\AppData\Local\pip\Cache\wheels\d9\c2\27\ac898adb26725eacef0139c486d270c641d86882743e16c2e6
Successfully built triegex
Installing collected packages: triegex
Successfully installed triegex-0.0.2
```

But I could not import it:

```
>>> import triegex
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
ModuleNotFoundError: No module named 'triegex'
```

My initial guess is that it is related to the `packages` argument of `setuptool.setup`, but I have not tested it. Perhaps you should use [`find_packages`](http://setuptools.readthedocs.io/en/latest/setuptools.html#using-find-packages) or add them manually?

**Comment by ZhukovAlexander** [comment]
Thanks for reporting this. Should be fixed in 7f336eaa889aca0eca3c1b385aac1ad670c2e213. Just install version ```triegex>=0.0.3```.

**Comment by 5j9** [comment]
Is working now, thanks!
